apps/site/assets/css/_link-block.scss

```
@@ -32,4 +32,12 @@
   &__inner-link {
     pointer-events: all;
   }
+
+  &__view-schedule {
+    position: relative;
+    bottom: 2px;
+    font-weight: 500;
+    font-size: 14px;
+    margin-left: 25px;
```

```suggestion
    margin-left: 2ch;
```

The` 25px` looks a good deal bigger than the spacing in the mock to me (I realize the mock doesn't really enumerate). But maybe this will be better? (I happen to enjoy `ch` units for spacing between letters but feel free to adjust).